src/components/footer/footer.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import Link from '@material-ui/core/Link';
+import { makeStyles } from '@material-ui/core/styles';
+import Container from '@material-ui/core/Container';
```

Comment:
if you want for @material-ui you can just write `import {Grid, Typography, Link, makeStyles, Container} from '@material-ui/core';`. That way you just have put down what your using and not have to type out the whole line. For example let say you wanted to add a loading icon you just add 'CircularProgress' to the line above instead typing out `import CircularProgress from '@material-ui/core/CircularProgress`